src/mlpack/prereqs.hpp

```diff
@@ -34,10 +34,6 @@
 #include <mlpack/core/cereal/template_class_version.hpp>
 #include <mlpack/core/data/has_serialize.hpp>
 
-// All code should have access to logging.
-#include <mlpack/core/util/log.hpp>
-#include <mlpack/core/util/io.hpp>
-
```

Comment:
These are already defined in `core.hpp` so they are useless here, and they are pulling load_image too early when it is not needed.
I just removed them it should be fine

Reply:
I think that's ok, so long as it doesn't make us include core.hpp where we could previously only include prereqs.hpp.